### PR TITLE
itk.meshread, itk.meshwrite

### PR DESCRIPTION
**Pull request by thewtex**
Addresses #1162

### CODE REVIEW

File: Wrapping/Generators/Python/itkTemplate.py [hunk 1]
```diff
@@ -543,14 +543,82 @@ def firstIfList(arg):
...
+            if not os.path.isfile(inputFileName):
+                msg += ("\nThe file doesn't exist. \n" +
+                       "Filename = %s" % inputFileName)
+            raise RuntimeError("Could not create IO object for reading file %s" % inputFileName + msg)
+        # Read the metadata from the mesh file.
+        meshIO.SetFileName(inputFileName)
+        meshIO.ReadMeshInformation()
+        dimension = meshIO.GetPointDimension()
+        componentAsString = meshIO.GetComponentTypeAsString(meshIO.GetPointPixelComponentType())
+        _io_component_type_dict = {
```
Comment (dzenanz) [inline Wrapping/Generators/Python/itkTemplate.py]: This is a duplicated definition, and therefore prone to discrepancies. Can this be shared with `_NewImageReader`'s definition?

**Comment by maekclena** [comment]
The failing test should be removed since `itk.MeshFileReader.New` now requires a `FileName` argument. See https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Mesh/wrapping/test/CMakeLists.txt#L13.

File: Wrapping/Generators/Python/itkTemplate.py (hunk 1 above)
Comment (thewtex) [reply Wrapping/Generators/Python/itkTemplate.py]: There is a subtle difference between the definitions -- to keep it cleaner they were defined this way. I added a comment to explain that it is intentional and what was done.

**Comment by thewtex** [comment]
> The failing test should be removed since `itk.MeshFileReader.New` now requires a `FileName` argument. See https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Mesh/wrapping/test/CMakeLists.txt#L13.

Good catch!

**Comment by thewtex** [comment]
@dzenanz @maekclena thank you for the reviews!

**Pull request merged**